src/test/java/edu/kit/datamanager/repo/util/validators/DOIValidatorTest.java

```diff
@@ -114,7 +109,7 @@ public void invalidHandleScheme() {
     @Test
     public void invalidURL() {
         try {
-            assertFalse(validator.isValid("doi.handle/10.1038/nphys1170"));
+            assertFalse(validator.isValid("doi.example/10.1038/nphys1170"));
```

Comment:
In your case code is also fine if isValid throws an OutOfMemoryException which should not happen!
To test for an explicit exception do it like this. (If you don't know which exception will be thrown something
goes wrong.)
@Test(expected = BadArgumentException.class)
public void invalidURL() {
validator.isValid(...)
}